modules/installation-openshift-local.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * getting_started/openshift-overview.adoc
+// * installing/index.adoc
+
+:_content-type: CONCEPT
+[id="installation-openshift-local_{context}"]
+= {product-title} Local Overview
+
+Red Hat {product-title} Local supports rapid application development to get started building {product-title} clusters. {product-title} Local is designed to run on a local computer to simplify setup and testing, and to emulate the cloud development environment locally with all of the tools needed to develop container-based applications. 
+
+Whatever your programming language, {product-title} Local hosts your application and brings a minimal, preconfigured Red Hat {product-title} 4.1 or newer cluster to your local PC without the need for a server-based infrastructure.
```

Comment:
```suggestion
Regardless of the programming language you use,  {product-title} Local hosts your application and brings a minimal, preconfigured {product-title} 4.1 or newer cluster to your local PC without the need for a server-based infrastructure.
```

Not a huge fan of "Whatever." There's only a few instances of it in the docs (26). Feel free to play with the wording, or ignore my suggestion altogether.

Reply:
Thanks for bringing this up! I decided to do away with the product version (4.1) for the sake of clarity. I think that the sentence still conveys that OpenShift Local brings a preconfigured OpenShift cluster to a local device. Feel free to correct me if I'm wrong. 

Just another note that I changed the wording from {product-title} Local to "OpenShift Local" because the former translated to "OpenShift Container Platform Local" and that terminology is not used in its marketing communication. Just to keep terms consistent, I decided to only include {product-title} when referring to clusters. Let me know too if this seems incorrect, or if you have any questions. 

Thanks for your time on this. :)